sandbox/apparmor/apparmor.go

```diff
@@ -320,10 +320,13 @@ func probeParserFeatures() ([]string, error) {
 	if err != nil {
 		return []string{}, err
 	}
-	features := make([]string, 0, 1)
+	features := make([]string, 0, 2)
 	if tryAppArmorParserFeature(parser, "change_profile unsafe /**,") {
 		features = append(features, "unsafe")
 	}
+	if tryAppArmorParserFeature(parser, "network qipcrtr dgram,") {
+		features = append(features, "qipcrtr-socket")
```

Comment:
Well, this won't be considered a "user-visible" API just since it's exposed via `snap debug`, since `snap debug` is a debug command and we don't guarantee backwards compatibility of debug commands.